test/functional/suite_test.go

```diff
@@ -243,4 +245,35 @@ var _ = BeforeEach(func() {
 	DeferCleanup(th.DeleteNamespace, namespace)
 	//Let's create the osp-secret in advance (in common to all the test cases)
 	DeferCleanup(k8sClient.Delete, ctx, CreateGlanceSecret(glanceName.Namespace, SecretName))
+
+	// Create MariaDBAccount and Secret.
+	// both controllers rely on reading the values from these objects.
+	// TODO: this set of features should likely be in mariadb-operator/api/test/crd.
```

Comment:
Also, I'm wondering how we're thinking about this in the long term: keep in mind that both `functional` tests and `kuttl` are not expecting a full deployed `OpenStackControlPlane`, as only the service operator CRs are generated (and the minimal amount of dependencies (mariadb/galera, rabbit, keystone, infra components).
I assume we can have an entry in `install_yamls` that generates the CRs expected as output from the openstack-operator, but I see that right now we're not there yet.
@zzzeek should we capture this conversation somewhere outside of this PR?

Reply:
for functional test we already have `SimulateMariaDBAccountCompleted` in mariadb-operator/api/helper but we can add there a function that create a MariaDBAccount

Reply:
ive created a helper in mariadb-operator and moved this to be local to the glanceapi_controller_test only where this doesn't otherwise happen.  for glance_controller_test the "ensuremariadbaccount" step is validated